### PR TITLE
Fix unnecessary highlighting of selected element

### PR DESCRIPTION
**Pull request by CarlSchwan**
Use focus-visible to only show focus ring on keyboard navigation

Fix #34589

### CODE REVIEW

**Review by artonge — APPROVED** [review]

File: apps/files/css/files.scss [hunk 1]
```diff
@@ -504,7 +504,7 @@ table td.selection {
 		padding: 16px;
 	}
 
-	&:focus + label {
+	&:focus-visible + label {
```
Comment (szaimen) [inline apps/files/css/files.scss:507]: does this work in chrome? I had the impression that it doesnt...

Comment (Pytal) [reply apps/files/css/files.scss:507]: Looks good https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#browser_compatibility ?

**Review by Pytal — APPROVED** [review]

File: apps/files/css/files.scss (hunk 1 above)
Comment (szaimen) [reply apps/files/css/files.scss:507]: @Pytal this is not what I meant. Did you test it and verified that it works equally on chrome and fierefox?

Comment (CarlSchwan) [reply apps/files/css/files.scss:507]: It works on chrome, just tested.

**Pull request merged**